Microsoft.Toolkit.Uwp.UI.Controls/OrbitView/OrbitViewItem.cs

```diff
@@ -0,0 +1,104 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// A class that can be used to provide data context for <see cref="OrbitView"></see> items by providing distance and diameter values
+    /// </summary>
+    public class OrbitViewItem : DependencyObject
+    {
+        /// <summary>
+        /// Gets or sets a value indicating the distance from the center.
+        /// Expected value betweeen 0 and 1
+        /// </summary>
+        public double Distance
+        {
+            get { return (double)GetValue(DistanceProperty); }
+            set { SetValue(DistanceProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="DistanceProperty"/> property
+        /// </summary>
+        public static readonly DependencyProperty DistanceProperty =
+            DependencyProperty.Register("Distance", typeof(double), typeof(OrbitViewItem), new PropertyMetadata(0.5));
```

Comment:
Also please use nameof(Distance) instead of "Distance" also please apply to the rest of the file :)